packages/netlify-cms-widget-markdown/src/MarkdownControl/Toolbar.js

```diff
@@ -114,52 +115,59 @@ export default class Toolbar extends React.Component {
       hasMark = () => {},
       hasInline = () => {},
       hasBlock = () => {},
+      editorComponents,
       t,
     } = this.props;
+    const isVisible = this.isVisible;
+    const showEditorComponents = !editorComponents || editorComponents.size > 1;
+    const showPlugin = ({ name }) => (editorComponents ? editorComponents.includes(name) : true);
```

Comment:
I had to change this line to `const showPlugin = ({ id }) => (editorComponents ? editorComponents.includes(id) : true);` to get it to work on my dev env.

Reply:
Good catch, not sure how this was working for me.